lib/src/generators/client.dart

```diff
@@ -491,7 +491,7 @@ class $clientName {
     if (operation.id != null) {
       methodName = operation.id!.camelCase;
     } else {
-      final cleanPath = path.replaceAll(RegExp(r'[{}]'), '');
+      final cleanPath = path.replaceAll(RegExp(r'\W'), '');
```

Comment:
Was the previous solution in #9 not robust enough? Curious what case you hit?

Reply:
However, I've just realized that that is breaking the conversion to camel case, as it is also removing the `/`. Let me rethink it.

Reply:
Replacing invalid characters with `_` should do the trick as the `.camelCase` will remove the `_` and make sure to capitalize the following valid character. What do you think?

Reply:
> What do you think?

Yes, as long as we can confirm that `camelCase` is followed at some point after the regex replace, I think this is good. Let `camelCase` do that cleanup.